+bids/query.m

```diff
@@ -23,11 +23,16 @@
 opts = parse_query(varargin);
 
 switch query
+<<<<<<< HEAD
```

Comment:
Looks like you got a couple unresolved merge conflicts here.

Reply:
bummer!  There should not have been any code changes in here; I suspect that I packed this on top of other changes.  

I will make a new PR with only changes to the README file.